### ISSUE TITLE
⭐ 02 - Recipe Generator (Hacktoberfest Demo)

### ISSUE THREAD

**Issue by NathanTarbert** [CopilotKit/CopilotKit#603]
### 🤩 Thank you for participating in the CopilotKit Hacktoberfest event. 

Follow [This Link to Get Started!](https://github.com/CopilotKit/CopilotKit/tree/main/community/content)🎉

#### NOTE: CopilotKit will allow multi-assign issues

  Please show your support and give a ⭐ to the CopilotKit repository.

### Description
Create a demo that suggests recipes based on available ingredients, using CopilotKit for natural language processing

#### Using CopilotKit
- **Required**: Your project must utilize CopilotKit as a core component.
- **Installation**: Include CopilotKit by following the installation guide at [CopilotKit Quickstart](https://docs.copilotkit.ai/quickstart).
- **Documentation**: Familiarize yourself with CopilotKit's features through the [Official Documentation](https://docs.copilotkit.ai).

#### Styling with Shadcn-UI
- **Required**: All UI components in your project should be styled using Shadcn-UI.
- **Installation**: Install Shadcn-UI [documentation](https://ui.shadcn.com/docs/installation)

![Swag_2](https://github.com/user-attachments/assets/d120bf97-95af-40a6-8618-6fb6a9198696)

**Comment by sanketshinde3001** [comment]
Interested..

**Comment by NathanTarbert** [comment]
@sanketshinde3001, please follow the link at the top to get started!

**Comment by sanketshinde3001** [comment]
Yesssss..

**Comment by sanketshinde3001** [comment]
@NathanTarbert Can i use  only NextJS for this ? And where to add this recipe generator in repo.

**Comment by NathanTarbert** [comment]
@sanketshinde3001, yes you can use NextJS but you also have to include CopilotKit.

Follow [this link](https://github.com/CopilotKit/CopilotKit/tree/main/community/content) to get started

**Comment by AJBrownson** [comment]
@NathanTarbert 
I'd like to work on this

**Comment by devshah207** [comment]
Can I also contribute please??

**Comment by sanketshinde3001** [comment]
@AJBrownson @devshah207 Thanks , but i started to work on it.

**Comment by AJBrownson** [comment]
@sanketshinde3001 of course
But it is written up there:
 "NOTE: CopilotKit will allow multi-assign issues"

**Comment by NathanTarbert** [comment]
That is correct @sanketshinde3001, we are allowing multi-assign

**Comment by sanketshinde3001** [comment]
@NathanTarbert can i add multiple useCopilotAction for one chatbot ?

**Comment by sanketshinde3001** [comment]
@NathanTarbert Made PR for it. Please try it and give me feedback.

**Comment by kom-senapati** [comment]
> @NathanTarbert can i add multiple useCopilotAction for one chatbot ?

Yes you can

**Comment by RohittCodes** [comment]
Hey @NathanTarbert I'd like to work on it.

**Comment by sanketshinde3001** [comment]
@kom-senapati i added but not working properly.

**Comment by kom-senapati** [comment]
> @kom-senapati i added but not working properly.

In my demos, I have used useCopilotAction multiple times its working

**Comment by sanketshinde3001** [comment]
ohk brother.

**Comment by sanketshinde3001** [comment]
@kom-senapati Can you please review my PR . I want feedback on  it so that i can improve it.

**Comment by kom-senapati** [comment]
Overall its good but ui can be improved like font sizes and colors @sanketshinde3001

**Comment by sanketshinde3001** [comment]
@kom-senapati i'll improve it sure. Thanks for feedback.

**Comment by NathanTarbert** [comment]
Thanks, everyone for such great support for each other's projects!

**Comment by sanketshinde3001** [comment]
@NathanTarbert I made PR on this issue. So what is the procedure of merging them. Is all PR gonna merge or best one gonna merge ?

**Comment by NathanTarbert** [comment]
@sanketshinde3001, we are going to be setting time aside next week and start merging the projects that work. 

Then when Hacktoberfest ends we will begin judging them for swag. 

Feel free to make any updates once your PR is merged with a new PR.

**Comment by sanketshinde3001** [comment]
@NathanTarbert That's nice !

**Comment by RohittCodes** [comment]
@quest-bot embark

**Comment by sanketshinde3001** [comment]
@quest-bot embark

**Comment by Mayank77maruti** [comment]
Hey @NathanTarbert , Please assign this task to me.
Would love to complete it.
Thank you.

**Comment by AJBrownson** [comment]
@quest-bot embark
